bundles/org.openhab.binding.fronius/README.md

```diff
@@ -2,23 +2,22 @@
 
 This binding uses the [Fronius Solar API V1](https://www.fronius.com/en/solar-energy/installers-partners/technical-data/all-products/system-monitoring/open-interfaces/fronius-solar-api-json-) to obtain data from Fronius devices.
 
-It supports Fronius inverters and Fronius Smart Meter.
-Supports:
+It supports Fronius inverters, smart meters and Ohmpilot devices connected to a Fronius Datamanager 1.0 / 2.0, Fronius Datalogger or with integrated Solar API V1 support.
 
+Supported inverters include:
+
+- Fronius Galvo
 - Fronius Symo
 - Fronius Symo Gen24
-- Fronius Smart Meter 63A
-- Fronius Smart Meter TS 65A-3
-- Fronius Ohmpilot
```

Comment:
These are no longer supported?

Reply:
They are, I only decided to rephrase that section a bit, it now reads:

> It supports Fronius inverters, smart meters and Ohmpilot devices connected to a Fronius Datamanager 1.0 / 2.0, Fronius Datalogger or with integrated Solar API V1 support.
>
> Inverters with integrated Solar API V1 support include:
>
> - Fronius Galvo
> - Fronius Primo
> - Fronius Symo
> - Fronius Symo Gen24
> - Fronius Symo Gen24 Plus